### DISCUSSION TITLE
Reloading the web app gives a 404.

### DISCUSSION THREAD

**Discussion by MNickelson** [ente-io/ente#4345] [Q&A] [answered]
I am trying to switch from the dev version to a build version of the photos web app using docker/nginx to point to the apps/photos/out folder.

Everything seems to be working but the entire site seems to be based on http://\<ente>/gallery. And if I ever hit F5 or refresh the page I get a 404 Not Found. I can add `.html` to the end of gallery and the page will load and redirect back to http://\<ente>/gallery and give the same behavior of a 404 on refreshing.

I'm using the latest commit from git (92c476cb6f3cb2262673008ce5e2bfbc1c1279f6)

**Comment by mngshm** [comment]
What do you mean by http:///gallery ? I have never heard of this. Can't seem to understand what your issue exactly.

**Reply by MNickelson** [reply]
Sorry, I meant http://mydomain/gallery.

I forgot that greater-than and less-than get eaten by markdown if they're not escaped.

**Reply by MNickelson** [reply, marked as answer]
Huh, using the Dockerfile from [here](https://help.ente.io/self-hosting/guides/web-app) rather than using `yarn build:photos` and then pointing an nginx container at apps/photos/out seems to work.
